styleguide/screens/Design/Responsive.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import cx from 'classnames';
+import dedent from 'dedent';
+
+import { H, T, D, A } from '../../components/Scaffold/Scaffold';
+import Specimen from '../../components/Specimen/Specimen';
+
+import m from '../../../globals/modifiers.css';
+import css from './Responsive.css';
+import scaffoldCss from '../../components/Scaffold/Scaffold.css';
+
+const Responsive = () => (
+  <div>
+    <H level={ 1 }>Responsive Design/Development</H>
+    <T elm="blockquote" className={ cx(css.blockquote, m.mtr, m.largeI, m.demi) }>
+      The control which designers know in the print medium, and often desire in the web medium,
+      is simply a function of the limitation of the printed page. We should embrace the fact that
+      the web doesn’t have the same constraints, and design for this flexibility. But first, we
+      must 'accept the ebb and flow of things.'
+    </T>
+    <T elm="i">
```

Comment:
I think this should be a `<cite />` elm, as per [html5doctor's breakdown of block quotes and citation](http://html5doctor.com/cite-and-blockquote-reloaded/)